backend/compact-connect/README.md

```diff
@@ -153,6 +159,31 @@ some resources may be left behind as CloudFormation is designed to err on the si
 loss. You can identify any resources that weren't destroyed by watching the stack deletion from the AWS CloudFormation
 Console, then looking at the resources after its delete is complete, to look for any with a `Delete Skipped` status.
 
+## About Route53 hosted zones
+
+A Hosted Zone in Route53 represents a collection of DNS records for a particular domain and its subdomains, managed
+together. See the [Route53 FAQs for more](https://aws.amazon.com/route53/faqs/). When creating a hosted zone, you have
+to also configure the domain name registrar (be it AWS or some other vendor) to point to the name servers associated
+with your hosted zone, before the records in the zone will have any effect. When deploying this app, creating a hosted
+zone in the AWS account for the UI and API domains is part of the environment setup. If you use the common approach
+of having your test environments be a subdomain of your production environments (i.e. `compcatconnect.org` for prod
+and `test.compcatconnect.org` for test), you need to delegate nameserver authority from your production hosted zone
+(`compactconnect.org` in this example) to your test account's hosted zone (`test.compactconnect.org`). To do this, you
```

Comment:
My intuition would say that this should be the opposite. Can you expand?

Reply:
Hm. We may need to have a conversation to help me understand where your intuition doesn't align with this, before I try to clarify in the doc.

Reply:
Ok, this was a misunderstanding on my end. We're good.